### PR TITLE
Skeleton is created for the backend

### PR DESCRIPTION
**Pull request by anna-yeddi**
Added/Updated:
- Server with Ts.ED, incl. placeholders for models, controller, service
- Mongoose
- Config

### CODE REVIEW

File: server/src/services/NotesService.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Service } from '@tsed/common'
+import { MongooseService } from '@tsed/mongoose'
+
+// To retrieve instance of Mongoose.Connection
+
+@Service()
+export class NotesService {
```
Comment (korzio) [inline server/src/services/NotesService.ts:7]: I think it is not necessary to add service if you planning to have only default connection

File: server/src/models/NoteModel.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+// TODO: Edit model in accordance to the DB data
+
+enum Categories {
+  CAT1 = 'cat1',
+  CAT2 = 'cat2',
+}
+
+@Model()
+export class NoteModel {
```
Comment (korzio) [inline server/src/models/NoteModel.ts:27]: Is it a WIP model?

File: server/src/models/PayloadModel.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import { NoteModel } from './NoteModel'
+
+// TODO: Edit model in accordance to the DB and API data
+
+enum Categories {
+  CAT1 = 'cat1',
+  CAT2 = 'cat2',
+}
+
+export class PayloadModel {
```
Comment (korzio) [inline server/src/models/PayloadModel.ts:19]: What this model is responsible for?

**Review by korzio — APPROVED** [review]

File: server/src/models/PayloadModel.ts (hunk 3 above)
Comment (anna-yeddi) [reply server/src/models/PayloadModel.ts:19]: This is a boilerplate for the request payloads if they'll be different from the Note(s) model or include additional information. It's also TBD.

File: server/src/models/NoteModel.ts (hunk 2 above)
Comment (anna-yeddi) [reply server/src/models/NoteModel.ts:27]: It is. I'll update it accordingly with DB data when structured.

**Pull request merged**